ergo/metaculus.py

```diff
@@ -1,3 +1,26 @@
+"""
+This module lets you get question and prediction information from Metaculus
+and submit predictions, via the API (https://www.metaculus.com/api2/)
+
+**Example**
+
+In this example, we submit our own prediction based on the community prediction.
+
+.. doctest::
+
+    >>> import ergo
+    >>> metaculus = ergo.Metaculus(username="oughtpublic", password="123456", api_domain="www")
```

Comment:
We need to find a better solution than providing username and password in the code. Maybe read it from an environment variable and set up Travis to provide that environment variable.